### PR TITLE
Fixed the thing

### PR DESCRIPTION
**Pull request by qorg11**
I fixed a LOT of typos in this commit

### CODE REVIEW

**Comment by qorg11** [comment]
also
#hacktoberfest

**Review by justcool393 — APPROVED** [review]

**Review by reagentoo — APPROVED** [review]

**Review by KeepSOBP — APPROVED** [review]

**Review by TheToto — APPROVED** [review]

**Review by FMudanyali — APPROVED** [review]

**Review by exciteabletom — APPROVED** [review]

**Comment by ivanka2012** [comment]
Okay, but let's be realistic here. Not all of them are actually written by scum like RIAA who take down stuff that *vaguely* infringes them, there are actual copyright violations out there.
tldr, even DMCA has legit uses, like youtube-dl

**Comment by qorg11** [comment]
free speech flag

**Review by polarisfm — APPROVED** [review]
This is a much needed refactor. This should significantly improve performance for all GitHub users. Please merge as soon as possible.

**Review by imachug — APPROVED** [review]

**Review by L33Tech — APPROVED** [review]

**Comment by endearingyoungcharms** [comment]
> Okay, but let's be realistic here. Not all of them are actually written by scum like RIAA who take down stuff that *vaguely* infringes them, there are actual copyright violations out there.
> 
> tldr, even DMCA has legit uses, like youtube-dl

Hold on, he's got a point

**Review by martynhaigh — APPROVED** [review]

**Comment by jessephus** [comment]
Hi 👋🏽  We are closing this pull request, because we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to this repository. Feel free to keep the discussion going in this thread. Please just be mindful to keep the discussion respectful and consistent with [GitHub’s Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines).

We understand that you may have questions about this takedown. Although we do not normally monitor comments posted to this repository, our team is reading through the comments and questions in this and other recent threads to see where we may be able to clarify our policies or answer some common questions. Thanks.

P.S. - No... this pull request does not qualify for Hacktoberfest. 😂 Nice try, though.

**Comment by reagentoo** [comment]
@jessephus Feel free to clone youtube-dl branch from your repository https://github.com/github/dmca/tree/416da574ec0df3388f652e44f7fe71b1e3a4701f

**Comment by piranna** [comment]
> P.S. - No... this pull request does not qualify for Hacktoberfest.  Nice try, though.

Report and block, as I've already done in https://github.com/NodeOS/NodeOS/issues/442
